vaccine_feed_ingest/runners/al/arcgis/normalize.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
+# isort: skip_file
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from datetime import datetime
+from typing import Optional, List
+
+from vaccine_feed_ingest_schema import location
+
+from vaccine_feed_ingest.utils.validation import BOUNDING_BOX
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("al/arcgis/normalize.py")
+
+STATE = "AL"
```

Comment:
consider: `schema.State.ALABAMA` ([from here](https://github.com/CAVaccineInventory/vaccine-feed-ingest-schema/blob/0a020a0e101d5c97ecd06a9ace15a4d6d798be2d/vaccine_feed_ingest_schema/location.py#L93)). As a general point for _future_ normalizers - consider using the `enums` where possible to avoid typos/etc.